src/constructors.jl

```diff
@@ -58,6 +58,13 @@ function Arb(str::AbstractString; prec::Integer = DEFAULT_PRECISION[])
     return res
 end
 
+function Arb(x::Rational; prec::Integer = DEFAULT_PRECISION[])
+    num = Arb(numerator(x); prec = prec)
+    denom = Arb(denominator(x); prec = prec)
+    div!(num, num, denom)
+    return num
+end
+
```

Comment:
Maybe Arb already optimizes division of integers but if not it might be beneficial to construct and `fmpq` from Flint and then set the value using that. But this is more work than I think is reasonable to do at the moment.

Reply:
Yes, maybe I should have added a comment. This is definitely better, but I didn't want to figure out how this can be done. So I opted for at least a correct conversion :)